src/types/Theme.ts

```diff
@@ -17,3 +20,12 @@ export interface Theme {
 export interface PropTheme {
   theme: Theme;
 }
+
+export interface CutomThemes {
```

Comment:
i think this interface isn't used anywhere 🤔

Reply:
That interfaces is needed in types/Config.ts file line 13.

`export interface Config extends Env, Constants, CustomThemes {}`

This is needed because getConfig is using Config interface.

Reply:
got it